cognite/client/_api/simulators/routines.py

```diff
@@ -0,0 +1,192 @@
+            list_cls=SimulatorRoutineList,
+            resource_cls=SimulatorRoutine,
+            method="POST",
+            filter=filter.dump() if isinstance(filter, CogniteFilter) else filter,
+            chunk_size=chunk_size,
+            limit=limit,
+        )
+
+    def list(
+        self,
+        limit: int = DEFAULT_LIMIT_READ,
+        filter: SimulatorRoutinesFilter | dict[str, Any] | None = None,
+        sort: CreatedTimeSort | None = None,
+    ) -> SimulatorRoutineList:
+        """`Filter simulator routines <https://developer.cognite.com/api#tag/Simulator-Routines/operation/filter_simulator_routines_simulators_routines_list_post>`_
+
+        Retrieves a list of simulator routines that match the given criteria
+
+        Args:
+            limit (int): Maximum number of results to return. Defaults to 1000. Set to -1, float(“inf”) or None to return all items.
+            filter (SimulatorRoutinesFilter | dict[str, Any] | None): The filter to narrow down simulator routines.
+            sort (CreatedTimeSort | None): The criteria to sort by.
+
+        Returns:
+            SimulatorRoutineList: List of simulator routines
+
+        Examples:
+            List simulator routines:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.routines.list()
+
+            Specify filter and sort order:
+                >>> from cognite.client.data_classes.simulators.filters import SimulatorRoutinesFilter
+                >>> from cognite.client.data_classes.simulators.routines import CreatedTimeSort
+                >>> res = client.simulators.routines.list(
+                ...     filter=SimulatorRoutinesFilter(simulator_integration_external_ids=["integration_ext_id"]),
+                ...     sort=CreatedTimeSort(order="asc")
+                ... )
+
+        """
+        self._warning.warn()
+        return self._list(
+            limit=limit,
+            method="POST",
+            url_path="/simulators/routines/list",
+            resource_cls=SimulatorRoutine,
+            list_cls=SimulatorRoutineList,
+            sort=[CreatedTimeSort.load(sort).dump()] if sort else None,
+            filter=filter.dump() if isinstance(filter, CogniteFilter) else filter,
+        )
+
+    @overload
+    def create(self, routines: Sequence[SimulatorRoutineWrite]) -> SimulatorRoutineList: ...
+
+    @overload
+    def create(self, routines: SimulatorRoutineWrite) -> SimulatorRoutineList: ...
+
+    def create(
+        self,
+        routines: SimulatorRoutineWrite | Sequence[SimulatorRoutineWrite],
+    ) -> SimulatorRoutine | SimulatorRoutineList:
+        """`Create simulator routine <https://developer.cognite.com/api#tag/Simulator-Routines/operation/create_simulator_routine_simulators_routines_post>`_
+        You can create an arbitrary number of simulator routines, and the SDK will split the request into multiple requests.
```

Comment:
no need to reveal implementation details: and the SDK will split the request into multiple requests.